solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -632,12 +613,6 @@ private void decorateRequest(Request req, SolrRequest<?> solrRequest, boolean is
     }
 
     setBasicAuthHeader(solrRequest, req);
-    for (HttpListenerFactory factory : listenerFactory) {
```

Comment:
No longer need to do this; the listeners are now at httpClient level, not request level.  Granted the listener onQueued can choose to register another listener (and one does do this).